doc/changelog/07-ssreflect/18224-changelog_18224.rst

```diff
@@ -0,0 +1,12 @@
+- **Changed:**
+  ssreflect no longer relies on the recovery mechanism
+  of the parsing engine coqpp, this can slightly change
```

Comment:
coqpp is not the parsing engine, it's the preprocessor for mlg files.

Reply:
I don't think we have a name for the parsing engine. In the code it lives in gramlib/ but that sounds too generic. It's a fork of camlp5 but isn't camlp5 anymore.

Reply:
That was indeed my assumption (coqpp = fork of (part of) camlp5). Anyway I don't care about the name, let's just remove it.